core/mount/mount.go

```diff
@@ -106,12 +106,12 @@ func (m *Mount) Mount(target string) error {
 	return m.mount(target)
 }
 
-// readonlyMounts modifies the received mount options
+// ReadonlyMounts modifies the received mount options
 // to make them readonly
-func readonlyMounts(mounts []Mount) []Mount {
+func ReadonlyMounts(mounts []Mount) []Mount {
```

Comment:
```suggestion
func MakeMountsReadonly(mounts []Mount) []Mount {
```

`ReadonlyMounts([]Mount) []Mount` sounds like as if it was a filter function

Reply:
Ah, good point! Actually, since the package is already `mount` - perhaps we this could be just `MakeReadonly`?